README.md

```diff
@@ -1,75 +1,49 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
```

Comment:
- [ ] OPTIONAL
I think you should give an uplift of your ReadMe by following the structure of this [recommended template](https://github.com/microverseinc/readme-template/blob/master/README.md).